engine/src/main/java/org/destinationsol/world/generators/SolarSystemGeneratorImpl.java

```diff
@@ -22,14 +22,34 @@
  *
  * This class also has access to the featureGenerators list from {@link SolarSystemGenerator}.
  * This allows it to choose which FeatureGenerators to use in populating the SolarSystem.
- * TODO: Define the behavior of default Planets in this class (As it is implemented in the game currently)
+ * TODO: Define the behavior of default SolarSystems in this class (As it is implemented in the game currently)
  */
 public class SolarSystemGeneratorImpl extends SolarSystemGenerator {
 
     @Override
     public void build() {
-        for (FeatureGenerator generator : featureGenerators) {
-            generator.build();
+        setPlanetCount(5);
+        setPossibleBeltCount(1);
+        setMazeCount(2);
+        setSunCount(1);
+        initializeRandomSunGenerators();
+        initializeRandomPlanetGenerators();
+        initializeRandomMazeGenerators();
+        initializeRandomBeltGenerators(.8f);
```

Comment:
These do not seem like they require a specific order, even though they do

Reply:
These actually do not have a specific order. Each one just get random Planet, Maze, Belt, or Sun types and adds instances of them to the activeFeatureGenerators list